src/qt/rpcconsole.cpp

```diff
@@ -1308,11 +1308,8 @@ void RPCConsole::unbanSelectedNode()
     {
         // Get currently selected ban address
         QString strNode = nodes.at(i).data().toString();
-        CSubNet possibleSubnet;
-
-        LookupSubNet(strNode.toStdString(), possibleSubnet);
-        if (possibleSubnet.IsValid() && m_node.unban(possibleSubnet))
-        {
+        std::optional<CSubNet> possibleSubnet = LookupSubNet(strNode.toStdString());
+        if (possibleSubnet.has_value() && m_node.unban(possibleSubnet.value())) {
```

Comment:
Hmm, `possibleSubnet.has_value()` will always be `true`. I guess this is an artifact from a previous incarnation of this PR that used `std::optional`.

Reply:
Yea, nice find, thanks!